autoload/fzf/vim.vim

```diff
@@ -640,15 +640,15 @@ function! s:sort_buffers(...)
   return b1 < b2 ? 1 : -1
 endfunction
 
-function! s:buflisted_sorted()
+function! fzf#vim#_buflisted_sorted()
```

Comment:
There is still `s:buflisted_sorted` in the code.

Reply:
:+1: I think I catched them all (with git grep).